### PR TITLE
Add a "features" parameter.

### PR DESCRIPTION
**Pull request by sunfishcode**
Add a "features" parameter that allows features to be passed to wit-bindgen's `--features` flag.

@yoshuawuyts @cdmurph32

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -40,7 +44,8 @@ runs:
       run: |
         set -ex
         for world in ${{ inputs.worlds }}; do
-          wit-bindgen markdown ${{ inputs.directory }} -w $world --check --html-in-md
+          ${parameter:+word}
+          wit-bindgen markdown ${{ inputs.directory }} -w $world --features ${{ inputs.features }}, --check --html-in-md
```
Comment (yoshuawuyts) [inline action.yml]: Can the list of features be empty? So like `--features=""` won't fail but resolves to no features?

**Review by yoshuawuyts — APPROVED** [review]
Just one question, but assuming that checks out this looks good!

File: action.yml (hunk 1 above)
Comment (sunfishcode) [reply action.yml]: The trailing comma in `${{ inputs.features }},` means an empty list will result in passing `--features ,`, which wit-bindgen interprets as an empty list.

Comment (sunfishcode) [reply action.yml]: I guess `--features=${{ inputs.features }}` would be a clearer way to achieve that. I've now changed it to do that.

**Comment by cdmurph32** [comment]
Awesome. Thanks!

**Comment by sunfishcode** [comment]
This is now tagged as v21.

**Pull request merged**